src/tabs/tabs.android.ts

```diff
@@ -103,7 +103,7 @@ function initializeNativeClasses() {
 
             // Get view as bitmap and set it as background. This is workaround for the disapearing nested fragments.
             // TODO: Consider removing it when update to androidx.fragment:1.2.0
-            if (hasRemovingParent && this.owner.selectedIndex === this.index) {
+            if (hasRemovingParent && this.owner.selectedIndex === this.index && this.owner.nativeViewProtected) {
```

Comment:
Did you try without it?
I can say for sure that this is not needed in my fork. I removed all those "tricks" from my fork but i dont really know the state in N

Reply:
on some rare occasions it was crashing because the nativeview was null, most likely because sometimes this view was being created/removed dynamically so there was nothing to "cache" in a drawable